lib/provider.coffee

```diff
@@ -92,12 +98,18 @@ module.exports =
       hasScope(previousScopesArray, 'entity.other.attribute-name.id') or
       hasScope(previousScopesArray, 'entity.other.attribute-name.parent-selector.css') or
       hasScope(previousScopesArray, 'entity.name.tag.reference.scss') or
-      hasScope(previousScopesArray, 'entity.name.tag.scss')
+      hasScope(previousScopesArray, 'entity.name.tag.scss') or
+      hasScope(previousScopesArray, 'entity.name.tag.reference.postcss') or
+      hasScope(previousScopesArray, 'entity.name.tag.postcss')
 
     isAtBeginScopePunctuation = hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.css') or
-      hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.scss')
+      hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.scss') or
+      hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.postcss') or
```

Comment:
Yeah, I know, `punctuation.section.property-list.begin.bracket.curly.postcss` is not currently used by language-post css, but I thought that probably it's the scope it *should* use, so I left it to be future-proof. Also language-css used `punctuation.section.property-list.begin.` in the past before atom 1.15 and then deprecated it.

However I can remove it, let me know :)